packages/cli/src/commands/dbCommands/save.js

```diff
@@ -30,7 +30,7 @@ export const handler = async ({
           name.length && `--name "${name}"`,
           '--create-db',
           '--experimental',
-          schema && `--schema=${schema}`,
+          schema && `--schema=\"${schema}\"`,
```

Comment:
Curious about the backslashes here, was it required, or just a test?

Reply:
I have to fix the tests and will double check those.

Reply:
Have added tests and address save and introspect.

Now save command succeeds:

```
yarn rw db save       
yarn run v1.22.10
$ '/Users/username/Dropbox (Personal)/projects/redwoodjs/pectin/node_modules/.bin/rw' db save
Creating database migration... [started]
$ '/Users/username/Dropbox (Personal)/projects/redwoodjs/pectin/node_modules/.bin/prisma' migrate save --name migration --create-db --experimental '--schema=/Users/username/Dropbox (Personal)/projects/redwoodjs/pectin/api/db/schema.prisma'
Prisma Schema loaded from db/schema.prisma
📼  migrate save --name migration

New datamodel:

datasource DS {
  // optionally set multiple providers
  // example: provider = ["sqlite", "postgresql"]
  provider = "sqlite"
  url = "***"
}

generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native", "rhel-openssl-1.0.x"]
}

// Define your own datamodels here and run `yarn redwood db save` to create
// migrations for them.
// TODO: Please remove the following example:
model UserExample {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
}


Prisma Migrate just created your migration 20201119181258-migration in

migrations/
  └─ 20201119181258-migration/
    └─ steps.json
    └─ schema.prisma
    └─ README.md

Run yarn prisma migrate up --experimental to apply the migration

Creating database migration... [completed]
✨  Done in 6.35s.
```